tests/integration/test_integration.py

```diff
@@ -83,3 +83,13 @@ async def test_build_and_deploy(ops_test: OpsTest):
     prometheus_k8s_unit = ops_test.model.units[f"{PROMETHEUS_APPLICATION_NAME}/0"]
     prometheus_k8s_private_address = prometheus_k8s_unit.data["private-address"]
     validate_scrape_target_is_added_to_prometheus(prometheus_k8s_private_address)
+
+
+
+async def test_remove_relation(ops_test: OpsTest):
+    await ops_test.model.applications["prometheus-edge-hub"].remove_relation("metrics-endpoint","prometheus-k8s")
+    try:
+        ops_test.model.wait_for_idle(apps=[APPLICATION_NAME], status='active', timeout=1000, raise_on_blocked=True)
```

Comment:
This charm **needs** a relation to work so here we would expect the charm to be in Blocked until a new relation is re-establishesd.

Reply:
We don't want to wait for the charm to be in "active" but in "blocked"